knockapi/resources/messages.py

```diff
@@ -0,0 +1,56 @@
+from .service import Service
+
+default_set_id = "default"
```

Comment:
What does this do? There seems to be a few places where we might've copied this over without actually using it.